Python/getversion.c

```diff
@@ -14,7 +14,12 @@ void _Py_InitVersion(void)
         return;
     }
     initialized = 1;
-    PyOS_snprintf(version, sizeof(version), "%.80s (%.80s) %.80s",
+#ifdef Py_GIL_DISABLED
+    const char *buildinfo_format = "%.80s experimental free-threading build (%.80s) %.80s";
```

Comment:
You have to update `static char version[250];` as well. I suggest to replace it with:

```
static char version[80*3 + 3 + strlen("experimental free-threading build ")];
```